server/controller/getForms.js

```diff
@@ -0,0 +1,13 @@
+const query = require('../database/queries/getForms');
+exports.getForms = (req, res) => {
+	const employeeID = req.params.employeeID;
```

Comment:
What if you get empty or undefined params? you have to check that before send it to the query!

Reply:
done !